Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Why don't we need to check also <code>"HomeTeam"</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>We can but it is not necessary because if she played in the league she sure played an away game (same for a home game).</p>

Reply:
<p>Thumbs up</p>